### ISSUE TITLE
Table 100% width does not allow dragging rows

### ISSUE THREAD

**Issue by jjspace** [kepano/obsidian-minimal#779]
### Is the bug present when using the default Obsidian theme?

No

### Is the bug present when snippets and plugins are disabled?

Yes

### Minimal theme version

7.7.16

### Describe the bug

When setting the table width to `100% pane width` it's not possible to drag table rows because the drag handlers end up under the gutter.

Digging into the console a bit it seems this is because `.table-100` removes the left padding that's necessary for the drag handlers on `cm-embed-block cm-table-widget markdown-rendered`. I believe `--table-drag-padding: var(--table-drag-space) 0;` needs to remove the last 0. when using it for `padding-inline-start` because that's [treating it as a percentage](https://developer.mozilla.org/en-US/docs/Web/CSS/padding-inline-start#values), not the left/right padding.

Edit: alternatively just change `padding-inline-start` to `var(--table-drag-space)` instead of `var(--table-drag-padding)`

### Debug info

<details><summary>Info</summary>
<p>

```
SYSTEM INFO:
	Obsidian version: v1.6.7
	Installer version: v1.4.13
	Operating system: #128-Ubuntu SMP Fri Jul 5 09:28:59 UTC 2024 5.15.0-118-generic
	Login status: logged in
	Catalyst license: none
	Insider build toggle: off
	Live preview: on
	Base theme: dark
	Community theme: Minimal v7.7.16
	Snippets enabled: 7
	Restricted mode: off
	Plugins installed: 34
	Plugins enabled: 30
		1: Editor Syntax Highlight v0.1.3
		2: Quick Switcher++ v4.4.0
		3: Maximise Active Pane v0.0.3
		4: Note Refactor v1.8.2
		5: Fullscreen mode plugin v0.1.2
		6: Show Whitespace v0.3.1
		7: Copy button for code blocks (fixed version manifest) v0.2.1
		8: Page Heading From Links v0.2.0
		9: Better Word Count v0.10.1
		10: Emoji Toolbar v0.4.1
		11: Hotkeys++ v0.2.7
		12: Paste URL into selection v1.7.0
		13: Git v2.25.0
		14: Natural Language Dates v0.6.2
		15: Linter v1.25.0
		16: Minimal Theme Settings v8.0.2
		17: Style Settings v1.0.8
		18: Dataview v0.5.67
		19: Tag Wrangler v0.6.1
		20: Hover External Link Plugin v1.0.0
		21: Simple Embeds v1.15.0
		22: Calendar v1.5.10
		23: Periodic Notes v0.0.17
		24: Emoji Shortcodes v2.2.0
		25: Auto Link Title v1.5.4
		26: Mind Map v1.1.0
		27: Link Favicons v1.8.4
		28: Optimize Canvas Connections v1.0.0
		29: Supercharged Links v0.12.1
		30: Advanced Tables v0.21.0

RECOMMENDATIONS:
	Custom theme and snippets: for cosmetic issues, please first try updating your theme and disabling your snippets. If still not fixed, please try to make the issue happen in the Sandbox Vault or disable community theme and snippets.
	Community plugins: for bugs, please first try updating all your plugins to latest. If still not fixed, please try to make the issue happen in the Sandbox Vault or disable community plugins.

```

</p>
</details>

**Comment by jjspace** [comment]
Related bug, the "Add new column" button seems to be hidden completely when using "Table 100% width". Might be due to some issues with the padding needed as well.

**Comment by kepano** [comment]
Yes it's intentional. When you make the table 100% width there is no longer space for the drag handles. You can still move rows using commands.

**Comment by jjspace** [comment]
@kepano why though? Is it really worth drastically reducing the functionality of tables for an extra 16-32 pixels? I definitely don't think so.

If you do then I really think it should be called out in the setting and [documentation](https://minimal.guide/tables) that this is intentional and not a bug because I was definitely scratching my head for a while trying to figure out why I couldn't drag rows anymore.

**Comment by kepano** [comment]
It's resolved in 7.7.18.
